Formula/juise.rb

```diff
@@ -24,12 +24,22 @@ class Juise < Formula
   depends_on "sqlite"
 
   def install
-    system "sh ./bin/setup.sh" if build.head?
+    system "sh", "./bin/setup.sh" if build.head?
+
+    # Prevent sandbox violation
+    inreplace "configure",
+      "SLAX_EXTDIR=\"`$SLAX_CONFIG --extdir | head -1`\"",
```

Comment:
I think they consider it a feature. That's a search path for extensions that `libslax` sets up for this purpose.